tests/unit/testRequestInformationContainer.js

```diff
@@ -14,177 +14,80 @@
  * limitations under the License.
  */
 
-var test = require('tape');
+var assert = require('assert');
 var RequestInformationContainer = require('../../src/classes/request-information-container.js');
 var Fuzzer = require('../../utils/fuzzer.js');
 
-test(
-  'Checking the RequestInformationContainer prototype after instantiation'
-  , function ( t ) {
-
-    var ric = new RequestInformationContainer();
-
-    t.deepEqual(
-      (typeof ric.setUrl)
-      , 'function'
-      , "setUrl should be of type function"
-    );
-
-    t.deepEqual(
-      (typeof ric.setMethod)
-      , 'function'
-      , 'setMethod should be of type function'
-    );
-
-    t.deepEqual(
-      (typeof ric.setReferrer)
-      , 'function'
-      , 'setReferrer should be of type function'
-    );
-
-    t.deepEqual(
-      (typeof ric.setUserAgent)
-      , 'function'
-      , 'setUserAgent should be of type function'
-    );
-
-    t.deepEqual(
-      (typeof ric.setRemoteAddress)
-      , 'function'
-      , 'setRemoteAddress should be of type function'
+describe('RequestInformationContainer', function () {
+  var f = new Fuzzer();
+  var cbFn, ric;
+  beforeEach(function () {ric = new RequestInformationContainer();});
```

Comment:
Despite the added redundancy I think it may be more clear to put this `beforeEach` hook in both of the nested `describe` blocks.